operator/option/config.go

```diff
@@ -249,6 +249,9 @@ const (
 	// Incoming traffic to http listener will return 308 http error code with respective location in header.
 	EnforceIngressHttps = "enforce-ingress-https"
 
+	// EnableIngressProxyProtocol enable proxy protocol for the listeners.
```

Comment:
```suggestion
	// EnableIngressProxyProtocol enable proxy protocol for all Ingress listeners. Note that _only_ Proxy protocol traffic will be accepted once this is enabled.
```
It's _really_ important to ensure that we call out that once you enable this, only Proxy protocol traffic will be accepted.

Reply:
@youngnick you are right. However, at least I would like to see similar behaviour that Traefik has (which means that you can define ip ranges from where it will accept proxy traffic. Basically Traefik is supporting both protocols: proxy and normal tcp at the same time). But that is something that is not supported by envoyproxy, so no can do in this repository.